### ISSUE TITLE
Support default fonts for text without font embedding

### ISSUE THREAD

**Issue by rototor** [rototor/pdfbox-graphics2d#5]
@FabioVassallo requires text rendered as fonts using the default PDF fonts without embedding any font file. His workaround/quickfix for that problem should be extended to handle more default font families correct.

```java
PdfBoxGraphics2DFontTextDrawer fontTextDrawer = new PdfBoxGraphics2DFontTextDrawer() {
           @Override
           protected PDFont mapFont(final Font font, final IFontTextDrawerEnv env) throws IOException, FontFormatException
           {
               return PDFFont.INTERNAL_PD_FONT_SANS_PLAIN;
           }
       };
```

This requires a test driver which uses the different default font families first.

**Comment by FabioVassallo** [comment]
Just a note: PDFFont is an internal class of mine, not intended to be exported.
PDFFont.INTERNAL_PD_FONT_SANS_PLAIN is just pointing to PDType1Font.HELVETICA



So in the example the overridden method should simply look like:



@Override

protected PDFont mapFont(final Font font, final IFontTextDrawerEnv env) throws IOException, FontFormatException

{

    return PDType1Font.HELVETICA;

}



@FabioVassallo<https://github.com/fabiovassallo> requires text rendered as fonts using the default PDF fonts without embedding any font file. His workaround/quickfix for that problem should be extended to handle more default font families correct.

PdfBoxGraphics2DFontTextDrawer fontTextDrawer = new PdfBoxGraphics2DFontTextDrawer() {

           @Override

           protected PDFont mapFont(final Font font, final IFontTextDrawerEnv env) throws IOException, FontFormatException

           {

               return PDFFont.INTERNAL_PD_FONT_SANS_PLAIN;

           }

       };

This requires a test driver which uses the different default font families first.

**Comment by rototor** [comment]
I've just released version 0.8 which provides a PdfBoxGraphics2DFontTextDrawerDefaultFonts class. Using that PdfBoxGraphics2dFontTextDrawer you always get some default font, but it tries to map the font style to the correct font. (i.e. Bold to PDType1Font.HELVETICA_BOLD and so on).

@FabioVassallo You might now use this one instead of your custom PdfBoxGraphics2dFontTextDrawer.
